torch/lib/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1476,7 +1497,8 @@ std::shared_ptr<ProcessGroup::Work> ProcessGroupNCCL::alltoall_base(
               comm,
               stream.stream());
         },
-        OpType::ALLTOALL_BASE);
+        OpType::ALLTOALL_BASE,
+        "all_to_all");
```

Comment:
Just to confirm, this won't include shape information for now right? That's fine for this diff but just wanted to make sure